### DISCUSSION TITLE
Recommended movies and shows vs. favorited movies and shows

### DISCUSSION THREAD

**Discussion by henrikfroehling** [trakt/trakt-api#410] [Q&A] [answered]
Just asking for clarification.
There are two new API endpoints [`movies/favorited/{period}`](https://trakt.docs.apiary.io/#reference/movies/favorited/get-the-most-favorited-movies) and [`shows/favorited/{period}`](https://trakt.docs.apiary.io/#reference/shows/favorited/get-the-most-favorited-shows).
The old ones, `movies/recommended/{period}` and `shows/recommended/{period}`, got removed, but are still working.
So, were the endpoints "recommended" just renamed to "favorited"?

**Comment by rectifyer** [comment, marked as answer]
Yes, they were renamed. The old methods still work to not break any apps, but you should move to the new endpoints to be the most up to date.
